src/renderer/corporate/dashboard/dashboard-container.jsx

```diff
@@ -89,12 +91,26 @@ class CorporateDashboardContainer extends PureComponent {
 					onAddDocument={this.handleAddDocument}
 					onEditDocument={this.handleEditAttribute}
 					onDeleteDocument={this.handleDeleteAttribute}
+					onAddMember={this.handleAddMember}
+					onOpenMemberDetails={this.handleOpenDetails}
+					selectedMember={this.state.selectedMember}
 					didComponent={<RegisterDidCardContainer returnPath={'/main/corporate'} />}
 				/>
 			</React.Fragment>
 		);
 	}
 }
 
+const mapStateToProps = (state, props) => {
+	const profile = identitySelectors.selectCorporateProfile(state);
+	return {
+		identity: identitySelectors.selectIdentity(state),
```

Comment:
profile should already include identity

Reply:
Good point.
Fixed on https://github.com/SelfKeyFoundation/Identity-Wallet/pull/1832/commits/82bcfef840644f20b6ebd7eb0cf8f7fe2e5f968d